assets/form.json

```diff
@@ -6,14 +6,14 @@
             "id": "years-in-college",
             "question": "How many years of college have you completed?",
             "type": "slider",
-            "range": [0, 5],
+            "range": [0, 4],
```

Comment:
RIT's engineering program is 5 years, which means 4.5 would be a possible options :penguin: